src/OpenAL/OpenToolkit.OpenAL/ALDevice.cs

```diff
@@ -0,0 +1,54 @@
+﻿//
+// ALDevice.cs
+//
+// Copyright (C) 2019 OpenTK
```

Comment:
*singing* iiiiiiiiin the yeeeeeeeeeeeeear twenty~~five~~twenty twenty~~five~~twenty......

Reply:
Fixed in latest